autoload/lsc/signaturehelp.vim

```diff
@@ -0,0 +1,57 @@
+if !exists('s:initialized')
+  let s:current_parameter = ''
+  let s:initialized = v:true
+endif
+
+augroup LscSignatureHelp
+  autocmd!
+  autocmd User LSCEditPreview execute 'match lscCurrentParameter /\V' . s:current_parameter . '/'
```

Comment:
I'm worried this will highlight matching text even if the preview window was opened by a hover request instead of a signature help request.

I think there are a couple ways we could solve that.

- Add a new utility like `lsc#util#once` that _doesn't_ use `<buffer>` when it sets up the command so that it can fire for the preview window.
- Instead of an autocommand add an optional argument to `lsc#util#displayAsPreview` which is a function to run while the buffer is still the preview window.